mysql/resource_rds_config_test.go

```diff
@@ -0,0 +1,206 @@
+package mysql
```

Comment:
I cannot run `make acceptance` here, so this doesn't really work - it's because of "acctest", but once you remove randomness, you won't need that, so I'm not sending the error message or trying to fix the issue on my PC.

Reply:
Also: please run "make acceptance" on your PC and make it pass - it's still not passing for me, because all of these tests are executed in all the cases.

You may want to add something like skip with TiDB: https://github.com/petoju/terraform-provider-mysql/blob/master/mysql/resource_ti_config_variable_test.go#L25

Reply:
So, since RDS MySQL doesn't change anything on the version string, I would need something like this to include some metadata to the version string:
```
+func serverVersionMetadataString(db *sql.DB) (string, error) {
+       var metadataVersionString string
+       err := db.QueryRow("SELECT @@GLOBAL.datadir").Scan(&metadataVersionString)
+       if err != nil {
+               return "", err
+       }
+
+       if strings.Contains(metadataVersionString, "rds") {
+               return "+rds", nil
+       }
+
+       return "", nil
+}
+
```

I will use this on my custom pre-check, but it can extend `serverVersionString` to include metadata (like: 5.7+rds, 8.0+TiDB, etc..) in the future.

Reply:
Here I was thinking about a more generic function that would return metadata do the Version (https://pkg.go.dev/github.com/hashicorp/go-version#section-readme). Possibly could be used by `serverVersion` or `serverVersionString`.

Reply:
I went with the boolean approach; it seemed simpler. The struct is also nice, but I think it should be implemented in `afterConnectVersion` (maybe on another PR). 

Now `make acceptance` and `make testrdsdb[58]` are all good.